tools/data/kinetics/README.md

```diff
@@ -26,12 +26,21 @@ Before we start, please make sure that the directory is located at `$MMACTION2/t
 
 ## Step 1. Prepare Annotations
 
-First of all, you can run the following script to prepare annotations.
+First of all, you can run the following script to prepare annotations by downloading from the official [website](https://deepmind.com/research/open-source/open-source-datasets/kinetics/).
 
 ```shell
 bash download_annotations.sh ${DATASET}
 ```
 
+Since some video urls are invalid, the number of video items in official annotations are less than the original one.
```

Comment:
the number of video items in current official annotations are less than the original official ones.